### DISCUSSION TITLE
install/air-tag-download/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1964] [Announcements]
# install/air-tag-download/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/air-tag-download/

**Comment by KubeKyrie** [comment]
export VERSION=v0.8.0      没有空格

**Comment by KubeKyrie** [comment]
一键安装执行脚本时，未指定${INSTALLER_VERSION}，其无默认值，执行脚本不报错
建议参数说明 INSTALLER_VERSION 默认值“无” ，且写明具体的有效值范围

**Reply by windsonsea** [reply]
目前页面已增加了参数说明：https://docs.daocloud.io/install/air-tag-download/#_3
